packages/editor/src/components/prefabs/PrefabEditors.tsx

```diff
@@ -55,6 +56,9 @@ export const PrefabIcon = ({ categoryTitle, isSelected }: { categoryTitle: strin
     Default: <IoCubeOutline className={`h-5 w-5 text-[${color}]`} />
   }
 
+  const IconElement = getState(PrefabShelfState).find((item) => item.category == categoryTitle)?.icon
```

Comment:
should this be reactive?

Reply:
No it's compile time only because of how we use vite with svgs